libcloud/dns/drivers/route53.py

```diff
@@ -315,7 +315,7 @@ def _update_multi_value_record(self, record, name=None, type=None,
 
         rrs = ET.SubElement(change, 'ResourceRecordSet')
         ET.SubElement(rrs, 'Name').text = record.name + '.' + \
-            record.zone.domain
+            record.zone.domain if record.name else record.zone.domain
```

Comment:
This is slightly confused / hard to read - would be easier if we used an actual if - I will make this change when merging the PR.